### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by injectives**

### CODE REVIEW

**Review by AndyHeap-NeoTech — APPROVED** [review]
It made my eyes go funny, but all looks ok to me.

File: build/license-header.txt [hunk 1]
```diff
@@ -1,16 +0,0 @@
-Copyright (c) "Neo4j"
-Neo4j Sweden AB [http://neo4j.com]
```
Comment (bigmontz) [inline build/license-header.txt]: Why was this file removed?

File: NOTICE.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Copyright (c) "Neo4j"
-Neo4j Sweden AB [http://neo4j.com]
+Neo4j Sweden AB [https://neo4j.com]
 
 This file is part of Neo4j.
```
Comment (bigmontz) [inline NOTICE.txt]: You might forgot to remove this phrase from this header.

File: bundle/NOTICE.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Copyright (c) "Neo4j"
-Neo4j Sweden AB [http://neo4j.com]
+Neo4j Sweden AB [https://neo4j.com]
 
 This file is part of Neo4j.
```
Comment (bigmontz) [inline bundle/NOTICE.txt]: You might forgot to remove this phrase from this header.

File: testkit-tests/NOTICE.txt [hunk 4]
```diff
@@ -1,5 +1,5 @@
 Copyright (c) "Neo4j"
-Neo4j Sweden AB [http://neo4j.com]
+Neo4j Sweden AB [https://neo4j.com]
 
 This file is part of Neo4j.
```
Comment (bigmontz) [inline testkit-tests/NOTICE.txt]: You might forgot to remove this phrase from this header.

**Review by bigmontz — APPROVED** [review]
📝

File: build/license-header.txt (hunk 1 above)
Comment (injectives) [reply build/license-header.txt]: Because it is no longer needed. The `license/neo4j_apache_v2/header.txt` supplied by the `org.neo4j.build:build-resources` will be used instead.
See: https://github.com/neo4j/neo4j-java-driver/pull/1496/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R472

File: NOTICE.txt (hunk 2 above)
Comment (injectives) [reply NOTICE.txt]: This looks as expected to me. It matches the `licensing/notice-apache2-prefix.txt` header supplied by the `org.neo4j.build:build-resources`.
Config: https://github.com/neo4j/neo4j-java-driver/pull/1496/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R507

File: bundle/NOTICE.txt (hunk 3 above)
Comment (injectives) [reply bundle/NOTICE.txt]: Same as here: https://github.com/neo4j/neo4j-java-driver/pull/1496#discussion_r1371809592

File: testkit-tests/NOTICE.txt (hunk 4 above)
Comment (injectives) [reply testkit-tests/NOTICE.txt]: Same as here: https://github.com/neo4j/neo4j-java-driver/pull/1496#discussion_r1371809592

**Review by AndyHeap-NeoTech — APPROVED** [review]
Good to go
